lib/ferrum/node.rb

```diff
@@ -62,6 +62,18 @@ def click(mode: :left, keys: [], offset: {}, delay: 0)
       self
     end
 
+    def set_attribute(value, name:)
```

Comment:
Before adding such a method I wanted first to make Ferrum/Cuprite API stable for this kind of methods, there's now a confusion with attributes, properties and `[](name)` see in Cuprite node.rb